Product/Production/Services/AuditRepositoryCore/src/test/java/gov/hhs/fha/nhinc/audit/transform/AuditTransformsTest.java

```diff
@@ -190,7 +190,7 @@ protected void testGetActiveParticipantSource(LogEventRequestType request, Boole
         }
 
         assertEquals("NetworkAccessPointID mismatch", ipOrHost, sourceActiveParticipant.getNetworkAccessPointID());
-        assertEquals("SourceActiveParticipant requestor flag mismatch", isRequesting,
+        assertEquals("SourceActiveParticipant requestor flag mismatch", Boolean.TRUE,
```

Comment:
I guess here's what it boils down to.  Should "sourceActiveParticipant.isUserIsRequestor()" be "true" here, even if "isRequesting" is "false"?